src/toil/test/provisioners/aws/awsProvisionerTest.py

```diff
@@ -124,6 +125,14 @@ def _test(self, spotInstances=False):
         self.sshUtil(runCommand)
         assert len(self.getMatchingRoles(self.clusterName)) == 1
 
+        checkStatsCommand = ['/home/venv/bin/python', '-c',
+                             'import json; import os; '
+                             'json.load(open("/home/" + [f for f in os.listdir("/home/") '
+                                                   'if f.endswith(".json")].pop()))'
```

Comment:
Mildly hacky since it leaves an open file descriptor but since the node is terminated immediately after this call I'm OK with it.